.goreleaser.yml

```diff
@@ -9,6 +9,7 @@ before:
   hooks:
     - go mod tidy
     - make manpages GH_VERSION={{.Version}}
+    - ./script/prepare-windows-cert.sh "{{.Env.GITHUB_CERT_PASSWORD}}" "{{.Env.DESKTOP_CERT_TOKEN}}"
```

Comment:
Is there a reason not to do this in the windows build section in a pre hook?

Reply:
there is!

the build matrix for windows (386 and amd64) means that the hook would be run twice in parallel. the curl would run twice and there are a pile of race conditions that occur as a result.